src/client/stylesheets/style.less

```diff
@@ -409,3 +409,10 @@ hr.wide {
 		margin-left: 0.2em;
 	}
 }
+
+@media only screen and (max-width: 451px){
+	#existing-edition-group-search-button{
+		font-size: calc(3px + 1.6vw);
```

Comment:
This makes it quite difficult to read. 
I would prefer if the button style was amended to allow a multiline text if there isn't enough space on one line.

Reply:
@MonkeyDo 


ya ! i think  multiline text is good idea .

Reply:
@MonkeyDo 
its as simple as that ,
the white-space property is set to "no-wrap"
that allows our text to ovreflow 


I HAVE set the property to "normal"
and wola ! it works!